integration-tests/http/package.json

```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "test:integration": "NODE_OPTIONS=--experimental-vm-modules jest --no-cache --maxWorkers=50% --bail --detectOpenHandles --forceExit --logHeapUsage",
+    "test:integration": "NODE_OPTIONS=--experimental-vm-modules jest --no-cache --maxWorkers=50% --bail --detectOpenHandles --forceExit --logHeapUsage -- __tests__/product/store/product.spec.ts",
```

Comment:
I guess this isn't intentional :) If it was about the flaky test, it should be fixed now, but let me know if it happens again.

Reply:
Definitely not 👌 thanks I missed it in my review